### ISSUE TITLE
_flexbox.less doesn't properly handle all -ms prefixes

### ISSUE THREAD

**Issue by benjaminapetersen** [openshift/origin#7559]
Per convesation with @sg00dwin yesterday~
Example:

```less
align-items: flex-start; 
-webkit-align-items: flex-start; 
-ms-align-items: start;   // bah, deviant!
```
But our mixin assumes `-ms-` is sufficient:
```less
.align-items(@align: stretch) {
  -webkit-align-items: @align;
     -moz-align-items: @align;
      -ms-align-items: @align;
          align-items: @align;
}
```
Grunt autoprefixer should handle this, though deprecated.  [PostCSS](https://github.com/postcss/autoprefixer) seems to be taking its place as the supported plugin.

**Comment by benjaminapetersen** [comment]
What it does:  "PostCSS Autoprefixer utilizes the most recent data from Can I Use to add only necessary vendor prefixes.   It also removes old, unnecessary prefixes..."

**Comment by benjaminapetersen** [comment]
The grunt package, for reference https://github.com/nDmitry/grunt-postcss

**Comment by benjaminapetersen** [comment]
@spadgett @jwforres @rhamilto

**Comment by spadgett** [comment]
Are all of the prefixes still necessary or can we use just the unprefixed styles?

**Comment by benjaminapetersen** [comment]
They are not all necessary, but thats the beauty of the processor, it will fix for us as things change over time.

**Comment by jwforres** [comment]
@benjaminapetersen is this needed to fix something in 3.2 or can this wait for 3.3, its a big enough change that i would hesitate

**Comment by sg00dwin** [comment]
We could temporarily include the ie10 specific declarations for the following that ones that we currently use. 

``` justify-content```
```align-items ```

**Comment by benjaminapetersen** [comment]
@jwforres I think @sg00dwin can answer that question, I'm not sure the ranking on the bug severity, but I've been looking into possible fixes.

**Comment by jwforres** [comment]
I would prefer to do this if its something that needs to get fixed in 3.2,
not sure which bug this is for

**Comment by sg00dwin** [comment]
@jwforres this would fix a couple of position/alignment issues in the top header on IE10

Here's a pr with the additional declarations included. https://github.com/openshift/origin/pull/7572/files

**Comment by spadgett** [comment]
Closing sine the original issue has been addressed in #7572. Opened #7853 to adopt the autoprefixer.
